src/commands/tutor-commands/tutor-leaderboard.ts

```diff
@@ -0,0 +1,52 @@
+import { SlashCommandBuilder } from "discord.js";
+import { Command } from "../../interfaces/Command";
+import { createEmbed } from "../../utils/embeded";
+import { getTutorLeaderboard } from "../../utils/supabase";
+import { commandLog, sendError } from "../../utils/logs";
+
+const tutorleaderboardBody = (bodyContent: string[]): string => {
```

Comment:
**suggestion:** rename function
**comment:** `tutorleaderboardBody` tells me nothing on what it does, functions do things, they typically start with verbs

Reply:
updated & pushed